src/components/docs/Excerpt/styles.module.css

```diff
@@ -41,7 +46,7 @@
 
 @media desktop-breakpoint-query {
   .content {
-    margin: 0 55px;
+    margin: 0 30px 0 50px;
```

Comment:
Matching styleguide

Reply:
Ok, but please make sure that it visually matches. I found that sometimes I would put in the rules according to invision, but it would look different